dev-requirements.txt

```diff
@@ -1,5 +1,3 @@
-setuptools>=38.6.0
-twine>=1.11.0
-wheel>=0.31.0
-black
-flake8
+git+https://github.com/thedataincubator/expectexception.git@master
+# can't unpin this as then definition doesn't work properly...
```

Comment:
It's not possible to install from git in `setup.py`? Otherwise we could get rid of dev-requirements?

Reply:
unfortunately I don't think so (when I tried it blew up...)

Reply:
Alright, seems to be possible but difficult (`dependency_links`) -- so not worth fiddling around with, we could also put them directly into the Makefile to reduce the numbers of files and locations to check.

Reply:
not in a notebook

Reply:
Ah, ok. We could also easily make such a function ourselves... But I don't care as long as we can rely on `pypi` as soon as we release ;)
We then would also need to pinthe right version (e.g. with `Pint` the current release does not seem to properly override existing units...)

Reply:
sounds good